klaxer/sinks.py

```diff
@@ -106,9 +107,18 @@ def post_message(self, message):
 
 
 def debounce(text):
-    if '(x' in text:
-        old_amount = text.split('(x')[1].split(')')[0]
+    # Regex pattern for text ending with dup indicators (e.g. "(x2)")
+    pattern = '\(x\d+\)$'
```

Comment:
While this is a fairly cheap pattern to compile, maybe move this to the module level?